usecase/usecase-wrt-linux-tests/samples/AlertDialog/README.md

```diff
@@ -0,0 +1,9 @@
+## Usecase Design
+
+This sample demonstrates Web Notification feature basic functionalities, include:
+
+* Notification constructor, requestPermission/permission and onclose
+
+## Spec Link
+
+https://crosswalk-project.org/jira/browse/XWALK-4061

```

Comment:
?? Seems you are not doing for this, please update it.

Also please use "BUG=" in the commit message. See https://crosswalk-project.org/contribute/contributing-code.html